newdle/client/src/components/UserAvatar.js

```diff
@@ -14,11 +15,17 @@ const defaultArgs = [
 function UserAvatar({user: {email, name, initials}, className, withLabel, size}) {
   initials = initials.replace(/[/?#]/g, '');
   const uri = `https://ui-avatars.com/api/${initials}/${size}/${defaultArgs}`;
-
+  
```

Comment:
please configure your editor to remove trailing whitespace when saving :)

Reply:
It is my mistake, I was trying to set the uri to sth which has a response.